### ISSUE TITLE
Nostalgy for TB 115

### ISSUE THREAD

**Issue by klausatlas** [alainfrisch/nostalgy#111]
OpenSUSE introduced TB 115 yesterday and Nostalgy is nor working for this TB version. Will there be an new version of Nostalgy that works with TB 115?

**Comment by hi5** [comment]
Yes, see https://github.com/opto/nostalgy-xpi/ and https://github.com/opto/nostalgy-xpi/issues/

If you can support the developer that is even better https://github.com/opto/nostalgy-xpi/issues/210
